internal/database/database.go

```diff
@@ -485,6 +485,40 @@ func (d *Database) GetPressures(ctx context.Context, platform string, flavors ..
 	return pressures, nil
 }
 
+// GetAllPressures returns the pressure for all flavors on the specified platform.
+func (d *Database) GetAllPressures(ctx context.Context, platform string) (map[string]int, error) {
```

Comment:
see if we can modify the `GetPressures` function such that if there is no flavor provided, then we just fetch every flavors.